torchvision/csrc/io/image/cpu/exif.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
```

Comment:
License-wise it seems OK as OpenCV is  Apache2, but I think we'll have to copy/paste the copyright notice from https://github.com/opencv/opencv/blob/097891e311fae1d8354eb092a0fd0171e630d78c/modules/imgcodecs/src/exif.cpp#L1-L41